documentation/src/docs/deployment/deployment.md

```diff
@@ -15,33 +15,35 @@ This guide describes the options for running Pravega for development, testing an
 
 There are two modes for running Pravega.
 
-- Standalone - Standalone mode is suitable for development and testing Pravega applications. It can either be run from the source code, from the distribution package or as a docker container.
-- Distributed - Distributed mode runs each component separately on a single or multiple nodes. This is suitable for production in addition for development and testing. The deployment options in this mode include a manual installation, running in a docker swarm or DC/OS.
+- **Standalone:** Standalone mode is suitable for development and testing Pravega applications. It can either be run from the source code, from the distribution package or as a docker container. For more information please see [Running Pravega](http://pravega.io/docs/latest/deployment/run-local/#standalone-mode).
+
+- **Distributed:** Distributed mode runs each component separately on a single or multiple nodes. This is suitable for production in addition for development and testing. The deployment options in this mode include a manual installation, running in a docker swarm or DC/OS.
```

Comment:
Aren't we including Kubernetes here? We should add it here, no?

Reply:
Right, it should be listed there. Maybe replacing DC/OS?

Reply:
@fpj, @adrianmo Accepted suggestion